File: PageBook.MAUI/Models/NoteModel.cs
using SQLite;

namespace PageBook.Models;

public class Note
{
    [PrimaryKey]
    public string Id { get; set; }
    public string Title { get; set; }
    public string Content { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime ModifiedAt { get; set; }
    public string Tags { get; set; }
}

File: app/src/main/java/ru/maxgog/pagebook/models/NoteModel.kt
package ru.maxgog.pagebook.models

import androidx.annotation.NonNull
import androidx.room.ColumnInfo
import androidx.room.PrimaryKey
import androidx.room.Entity

@Entity(tableName = "notes")
data class NoteModel(
    @PrimaryKey(autoGenerate = true)
    @NonNull
    @ColumnInfo(name = "NoteId") val id: Int = 0,
    @ColumnInfo(name = "Title") val title: String = "",
    @ColumnInfo(name = "Content") val content: String = "",
    //@ColumnInfo(name = "Color") val color: Int = 0xFFBB86FC.toInt(),
    @ColumnInfo(name = "AtCreated") val createdAt: Long = System.currentTimeMillis()
)